live-examples/js-examples/date/date-toprimitive.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js" class="language-js">
+  const date = new Date('20 December 2019 14:48'); //initialize a new Date object
```

Comment:
All the lines have an indentation, which they should not have.

Also we don't generally use generic comments saying what each line it doing. We aim to write simple examples that can speak for themselves, and only use comments sparingly to point out particularly nonobvious things.